tests/CMakeLists.txt

```diff
@@ -237,10 +237,10 @@ if(WITH_PYTHON)
                restarting/from_6.3.13/DrUpgradeRestart-2.txt)
   add_fdb_test(
     TEST_FILES restarting/from_6.3.13/SnapCycleRestart-1.txt
-               restarting/from_6.3.13/SnapCycleRestart-2.txt IGNORE) # TODO re-enable once snap test failures are fixed!
```

Comment:
Is there a reason the restarting from 6.3 ones are still ignored? If they are intentionally ignored, it might be useful to have a comment on why.
Also could you link the PR(s) with the actual fixes to this PR and/or the issue for re-enabling the snapshot tests, for documentation purposes?

Reply:
Ah just saw the latest commit on disabling restarting from < 7.0

Reply:
The PR that fixes this issue is: https://github.com/apple/foundationdb/pull/6047. The motivation for removing the upgrades from old versions is that it requires backporting many testing changes all the way back to 6.3. A lot of the nightly failures we've seen have simply been because some testing change was not backported all the way to 6.2 and 6.3. 6.3 snapshotting will still have test coverage in the `release-6.3` nightly tests. Also, being able to upgrade from a snapshot is a nice feature to have, but not strictly necessary, since you can upgrade after restoring the snapshot.

Reply:
Ok, sounds good. Thanks for explaining!